rcon/steam_utils.py

```diff
@@ -67,234 +106,297 @@ def wrapper(steamd_id: str, *args, **kwargs):
     return decorator
 
 
-def get_steam_api_key() -> str | None:
-    global last_steam_api_key_warning
+@ttl_cache(60 * 60 * 12, cache_falsy=False, is_method=False)
```

Comment:
does this really need to be cacheable? I would say, as `fetch_steam_player_summary_mult_players` is cached already, we can safely omit this caching :)

Reply:
I concur